hadoop-project/pom.xml

```diff
@@ -1918,6 +1918,9 @@
       <activation>
         <activeByDefault>false</activeByDefault>
       </activation>
+      <properties>
+        <protoc.path>/opt/protobuf-3.7/bin/protoc</protoc.path>
+      </properties>
```

Comment:
Yeah, this should go away.

Reply:
This is kept temporarily to make jenkins run for this PR alone.
This PR alone needs both versions of protoc. Before patch 2.5.0 and after patch 3.7.1.
Dockerfile will be updated in subsequent PR once this PR goes in.